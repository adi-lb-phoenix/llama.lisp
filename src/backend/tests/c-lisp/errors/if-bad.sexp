(c-lisp
    (define ((print int) (n int)))

    (define ((main void))
        (if #t
            (call print 5)
            (call print 4)
            (call print 3))))
