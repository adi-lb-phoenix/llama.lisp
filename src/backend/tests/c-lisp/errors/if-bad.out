Error in statement: ['if', True, ['call', 'print', 5], ['call', 'print', 4], ['call', 'print', 3]]
CodegenError: Bad if statement: ['if', True, ['call', 'print', 5], ['call', 'print', 4], ['call', 'print', 3]]
